api/src/main/java/org/pmiops/workbench/utils/ComparableUtils.java

```diff
@@ -0,0 +1,28 @@
+package org.pmiops.workbench.utils;
+
+/**
+ * A few generic utilities to hopefully make Comparators less ugly
+ */
+public class ComparableUtils {
+
+  public static <T extends Comparable<T>> Boolean isLessThan(T one, T other) {
```

Comment:
I thought about putting the generic stuff at the class level, but was worried that would mean I have to qualify the type explicitly at the call site.